File: python/pad_array.py
#REMEMBER TO PSEUDOCODE
# get length of array
# if length of arr is less than value add either None as default value or value
# return new arr

def pad(array, min_size, value = None):
    if min_size <= len(array):
        return list(array)
    else:
        padding = [value] * (min_size - len(array))
        return list(array) + padding
    
    
    

File: js/padArray.js
// REMEMBER TO PSEUDOCODE
const pad = (array, minSize, value=null) => {
    if (minSize <= array.length) {
        return array
    }
    let newArr = [...array]
    let addSize = minSize - array.length
    for (let i = 0; i < addSize; i++) {
        newArr.push(value)
    }
    return newArr
}

exports.pad = pad;
